views/static/press-kit.jade

```diff
@@ -14,9 +14,10 @@ block extraHead
     }
 
 block content
-  h2 Press Kit
-  p Thanks for your interest in HabitRPG! The following images can be used for articles or videos about HabitRPG. For more information, please contact Siena Leslie at leslie@habitrpg.com.
-  p Download all image: <a class='btn btn-success' href='/presskit/presskit.zip'>presskit.zip</a>
+  h2= env.t('presskit')
+  p= env.t('presskitText')
+  p= env.t('presskitDownload')
```

Comment:
This is going to be real nitpicky, so I apologize in advance, but can you change this to:
```
p= env.t('presskitDownload')
  | &nbsp;
```

It'll add a space between the colon and the button.

![screen shot 2015-01-29 at 4 53 20 pm](https://cloud.githubusercontent.com/assets/2916945/5967891/5b7895ae-a7d7-11e4-8f7c-3b15b3f3ad33.png)